py_mind_memo/dialogs.py

```
@@ -79,3 +79,82 @@ def show(self):
+
+        tk.Label(main_frame, text="Select a template:").pack(anchor=tk.W, pady=(0, 5))
+
+        # Listbox for template names
+        self.listbox = tk.Listbox(main_frame, width=40, height=10, selectmode=tk.SINGLE)
+        self.listbox.pack(fill=tk.BOTH, expand=True)
+        
+        for tmpl in self.templates:
+            self.listbox.insert(tk.END, tmpl["name"])
+
+        # Double click to select
+        self.listbox.bind("<Double-Button-1>", lambda e: self.on_ok())
```

Ruff (0.15.9)

[warning] Unused lambda argument: `e`

(ARG005)